lib/collections.js

```diff
@@ -11,12 +11,16 @@ var paragraph =
 // Question 1
 // write code to split the above paragraph into an array of words
 // then normalize the words in the array
-
 // The word list normalized (all lower case with punctuation removed)
-var normalizedWords = [];
 
 /* your code starts here */
-
+var words = paragraph.split(/\s+/);
+var normalizedWords = [];
+for (var i = 0; i < words.length; i++) {
+  var string;
+  string = words[i].toLowerCase().replace(/[^\w']+/, "");
+  normalizedWords[] = string;
```

Comment:
This is not proper JavaScript syntax. The way to add an element to an array is to use `.push()`.